prog.go

```diff
@@ -20,6 +20,8 @@ import (
 // ErrNotSupported is returned whenever the kernel doesn't support a feature.
 var ErrNotSupported = internal.ErrNotSupported
 
+var errUnsatisfiedReference = errors.New("unsatisfied reference")
```

Comment:
The error string is nice and short, but still somewhat opaque. How about `unsatisfied reference to map`? It's currently only used for maps, so could be `errUnsatisfiedMapReference`.

Reply:
The error is actually `possible attempt to load program instead of collection: map foobar: unsatisfied reference`, so I'd leave it as it is.

Reply:
Okay, cool! That's perfect :+1: